aaelf64/aaelf64.rst

```diff
@@ -1249,6 +1249,10 @@ Finally the bit-field of X specified in the table (those bits of X picked out by
 
 For "``MOVW``" type relocations it is the assembler’s responsibility to encode the hw bits (bits 21 and 22) to indicate the bits in the target value that the immediate field represents.
 
+Relocation relaxation
```

Comment:
It seems that the term "linker optimization" is often used when the number of bytes does not change while "linker relaxation" is used when the number of bytes decreases.

The two arches which support GOT optimization use the term optimization.

Reply:
For the Mach-O directive `.loh`, it stands for Linker Optimization Hints :)